src/main/java/com/datastax/cdm/feature/WritetimeTTL.java

```diff
@@ -262,9 +259,7 @@ private OptionalLong getMaxWriteTimeStamp(Row row) {
 
     public Integer getLargestTTL(Row row) {
         if (logDebug)
-            logger.debug("getLargestTTL: customTTL={}, ttlSelectColumnIndexes={}", customTTL, ttlSelectColumnIndexes);
-        if (this.customTTL > 0)
-            return this.customTTL.intValue();
+            logger.debug("getLargestTTL: ttlSelectColumnIndexes={}", ttlSelectColumnIndexes);
```

Comment:
shouldn't this be printing `getCustomTTL().intValue()` here also?

Reply:
@msmygit No. When computing largest TTL, custom TTL should play no role as they are two separate items. When using custom TTL, it should be set on the target w/o impacting how largest TTL is computed. The same applies for writetime. Since the two logic were intertwined, it caused this bug.